client/src/main/java/com/alibaba/nacos/client/naming/remote/gprc/redo/RedoScheduledTask.java

```diff
@@ -75,6 +76,12 @@ private void redoForInstance(InstanceRedoData redoData) throws NacosException {
                 if (isClientDisabled()) {
                     return;
                 }
+                if (redoData instanceof BatchInstanceRedoData) {
```

Comment:
abstract the different redo logics to one method such as `redoRegister`.
And move from line 79 to 86 to new method.